Plugin.php

```diff
@@ -173,6 +173,16 @@ public function registerNotificationRules()
         ];
     }
 
+    public function registerReportWidgets()
+    {
+        return [
+            'RainLab\User\ReportWidgets\NewUsers' => [
+                'context' => 'dashboard',
+                'label' => 'rainlab.user::lang.widgets.new_users.add_widget',
```

Comment:
Why is `add_widget` a different key from `default_title` when they both have the same content?

Reply:
`add_widget` refers to how this widget is listed in the dashboard dropdown, and `default_title` is obviously the default title for the widget. I suppose they could piggy back off the same key, but it's being used in two fairly different contexts so I made them different keys. I did this because I never know what flexibility different languages need, I only speak English. I'm fine with changing the value of one, or consolidating them to a single key, just let me know which you would prefer.

If you choose to use the same key, what should it be called given that it's being used in two different places?

Reply:
I'm fine with using the `default_title` for both contexts as the add widget section refers to the default title of the widget most of the time.